Make `from_parts` a LRU to increase the chance we can preserve the internal cache (#1434)
aio-libs/yarl: CHANGES/1434.misc.rst

> Improved cache performance for operations that modify the :class:`~yarl.URL` object -- by :user:`bdraco`.

Author: ```suggestion
Improved cache performance for operations that produce a new :class:`~yarl.URL` object -- by :user:`bdraco`.
```
